dali/test/python/test_operator_python_function.py

```diff
@@ -204,6 +204,12 @@ def flip(image):
     return numpy.fliplr(image)
 
 
+def dlflip(image):
```

Comment:
Do you use it anywhere?

Reply:
Hmmm... I had it, but apparently it got lost somehow... strange.

Reply:
Tests added.